packages/dd-trace/src/config.js

```diff
@@ -486,11 +486,17 @@ class Config {
     const {
       AWS_LAMBDA_FUNCTION_NAME,
       DD_AGENT_HOST,
+      DD_AI_GUARD_ENABLED,
+      DD_AI_GUARD_ENDPOINT,
+      DD_AI_GUARD_TIMEOUT,
+      DD_AI_GUARD_MAX_MESSAGES_LENGTH,
+      DD_AI_GUARD_MAX_CONTENT_SIZE,
```

Comment:
we try to keep this in alphabetical order:

```suggestion
      DD_AI_GUARD_ENABLED,
      DD_AI_GUARD_ENDPOINT,
      DD_AI_GUARD_MAX_MESSAGES_LENGTH,
      DD_AI_GUARD_MAX_CONTENT_SIZE,
      DD_AI_GUARD_TIMEOUT,
```